docs/embed/configuration.md

```diff
@@ -32,10 +32,10 @@ Even though this is possible, it isn't recommended. Environment variables aren't
 
 #### Using REST APIs
 
-The recommended way is to load the data using a custom REST endpoint. Set the `CREDENTIALS_OVERWRITE_ENDPOINT` to a path under which this endpoint should be made available.
+The recommended way is to load the data using a custom REST endpoint. Set the `CREDENTIALS_OVERWRITE_ENDPOINT` to a path under which this endpoint should be made available. You can set `CREDENTIALS_OVERWRITE_ENDPOINT_AUTH_TOKEN` to require a token for accessing the endpoint. When this token is configured, the endpoint is only accessible if the token is included in the `Authorization` header as a `Bearer` token.
 
 /// note
-The endpoints can be called just one at a time for security reasons.
+The endpoints can be called just one at a time for security reasons, unless `CREDENTIALS_OVERWRITE_ENDPOINT_AUTH_TOKEN` is set.
```

Comment:
I'm not sure about the "one at a time".
It's rather that the endpoint will be effective only once per instance lifetime. If you want to re-override credentials, you need to restart the instance.
Do you agree @afitzek ?